Public/Src/Cache/ContentStore/Hashing/ContentHasher.cs

```diff
@@ -183,12 +179,13 @@ private sealed class HashingStreamImpl : HashingStream
             private static readonly byte[] EmptyByteArray = new byte[0];
             private static readonly Task<bool> TrueTask = Task.FromResult(true);
 
-            private static readonly Stopwatch _sw = Stopwatch.StartNew();
+            private static readonly Stopwatch Stopwatch = Stopwatch.StartNew();

```

Comment:
Please avoid instance names that are identical (here even identical casing) to type names. (This is one argument for not using PascalCase for field names.) Maybe timer?

Reply:
Agree. Renamed to Timer.